lib/transform.js

```diff
@@ -108,7 +108,7 @@ function determineTitle(title, notitle, lines, info) {
   return info.hasStart ? lines[info.startIdx + 2] : defaultTitle;
 }
 
-exports = module.exports = function transform(content, mode, maxHeaderLevel, title, notitle, entryPrefix, processAll, updateOnly) {
+exports = module.exports = function transform(content, mode, maxHeaderLevel, minHeaderLevel = 1, title, notitle, entryPrefix, processAll, updateOnly) {
```

Comment:
I'm not sure this matters, but it seems like this old codebase doesn't use default params. I'm thinking let's avoid them to not run afoul with compatibility questions.